moveit_core/collision_detection_fcl/src/collision_world_fcl.cpp

```diff
@@ -52,9 +52,11 @@
 
 #include <boost/bind.hpp>
 
+rclcpp::Logger LOGGER_COLLISION_WORLD = rclcpp::get_logger("collision_world.fcl");
+
 namespace collision_detection
 {
-const std::string CollisionDetectorAllocatorFCL::NAME("FCL");
+const std::string CollisionDetectorAllocatorFCL::NAME_("FCL");
```

Comment:
This has to match `CollisionDetectorAllocatorTemplate::NAME` please revert

Reply:
https://github.com/ros-planning/moveit2/pull/41/commits/18a607db2fc15a3840cbb8adafd1e858b9019ecc